balena/models/release.py

```diff
@@ -293,3 +293,131 @@ def finalize(self, commit_or_id):
             'release', 'PATCH', params=params, data=data,
             endpoint=self.settings.get('pine_endpoint')
         )
+
+    def set_is_invalidated(self, commit_or_id, is_invalidated):
+        """
+        Set the is_invalidated property of a release to True or False.
+
+        Args:
+            commit_or_id (str): release commit (str) or id (int).
+            is_invalidated (bool): True for invalidated, False for validated.
+
+        Returns:
+            OK
+
+        Raises:
+            ReleaseNotFound: if release couldn't be found.
+
+        """
+
+        id = self.get(commit_or_id)['id']
+
+        params = {
+            'filter': 'id',
+            'eq': id
+        }
+        
+        data = {
+            'is_invalidated': is_invalidated
+        }
+
+        return self.base_request.request(
+            'release', 'PATCH', params=params, data=data,
+            endpoint=self.settings.get('pine_endpoint')
+        )
+
+    def set_note(self, commit_or_id, note):
+        """
+        Set a note for a release.
+
+        Args:
+            commit_or_id (str): release commit (str) or id (int).
+            note (str): the note.
+
+        Returns:
+            OK
+
+        Raises:
+            ReleaseNotFound: if release couldn't be found.
+
+        """
+
+        id = self.get(commit_or_id)['id']
+
+        params = {
+            'filter': 'id',
+            'eq': id
+        }
+        
+        data = {
+            'note': note
+        }
+
+        return self.base_request.request(
+            'release', 'PATCH', params=params, data=data,
+            endpoint=self.settings.get('pine_endpoint')
+        )
+
+    def set_known_issue_list(self, commit_or_id, known_issue_list):
+        """
+        Set a known issue list for a release.
+
+        Args:
+            commit_or_id (str): release commit (str) or id (int).
+            known_issue_list (str): the known issue list.
+
+        Returns:
+            OK
+
+        Raises:
+            ReleaseNotFound: if release couldn't be found.
+
+        """
+
+        id = self.get(commit_or_id)['id']
+
+        params = {
+            'filter': 'id',
+            'eq': id
+        }
+        
+        data = {
+            'known_issue_list': known_issue_list
+        }
+
+        return self.base_request.request(
+            'release', 'PATCH', params=params, data=data,
+            endpoint=self.settings.get('pine_endpoint')
+        )
+
+    def set_release_version(self, commit_or_id, semver):
```

Comment:
@nghiant2710 sorry if I wasn't clear :disappointed:, I meant the method name:
```suggestion
    def set_release_semver(self, commit_or_id, semver):
```

Reply:
@thgreasi, it's called `release version` in the dashboard so I'm not sure if we should change the method name to `set_release_semver` as it should match with our dashboard
![image](https://user-images.githubusercontent.com/8970907/203352267-9b63a796-1289-4de2-a8da-7a8cba504914.png)

Reply:
Fair enough